packages/payments-plugin/e2e/mollie-payment.e2e-spec.ts

```
@@ -423,28 +323,33 @@ describe('Mollie payments', () => {
-                .reply(200, mockData.mollieOrderResponse);
+                .reply(200, mollieMockData.molliePaymentResponse);
             await addManualPayment(server, 1, 10000);
             await shopClient.query(CREATE_MOLLIE_PAYMENT_INTENT, {
                 input: {
-                    paymentMethodCode: mockData.methodCode,
+                    paymentMethodCode: mollieMockData.methodCode,
                 },
             });
             expect(mollieRequest.amount?.value).toBe('909.88'); // minus 100,00 from manual payment
             let totalLineAmount = 0;
             for (const line of mollieRequest?.lines) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)